src/net_processing.cpp

```diff
@@ -2121,29 +2131,33 @@ bool SendMessages(CNode *pto)
         //
         // Message: addr
         //
-        if (pto->nNextAddrSend < nNow)
+        if ((pto->nNextAddrSend < nNow) && pto->fInbound)
```

Comment:
The way that the we announce our presence to the network is by pushing out our own IP with ADDR. The code for this is [here](https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/release/src/net_processing.cpp#L348). 

AFAIK there is no special treatment for our own IP, so if you don't broadcast ADDR to outbound peers, you won't be able to announce yourself to the network. Thus you'll end up never getting any incoming peers.

Reply:
you're right, changes made... thanks!